### DISCUSSION TITLE
stop using ai. now.

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#248] [Admin General]
@nmsderp @Rubiidev-18 @rubiidev18alt @Rubiidev-IPad

please stop, the editor is running off of ai-generated code and rubii keeps adding stuff made with ai. **write your own code, _please_**

**Comment by dumorando** [comment]
from CONTRIBUTING.md<br />
Using AI to create commits is no longer permitted. If you need to use AI for anything here, we probably don't need your contribution. This applies to members of the Bark Team as well.

**Reply by dumorando** [reply]
fyi **we can tell when you use ai**

**Comment by mariocraft987** [comment]
If you do keep using AI and you are a Bark member, you will be removed from the repo

**Comment by nmsderp** [comment]
i barely use ai now, i now force myself to only use ai if absolutely needed (unless its css, but i have got better at css)

**Reply by raynepaws** [reply]
am i correct in saying the editor is powered by ai scripts though

**Reply by raynepaws** [reply]
also, ai is never *absolutely needed,* just ask for help!!

**Reply by dumorando** [reply]
> am i correct in saying the editor is powered by ai scripts though

literlly all of sketch.js lmao

<hr>

fair enough i havent seen you make many commits with ai in a while but we pinged u cause
![image](https://github.com/mariocraft987/bark-coding/assets/144301348/dd883154-318a-4771-a59d-509dbd7c08a5)

**Reply by dumorando** [reply]
theres another person though that makes a shit ton of ai commits and still does to this day so thats mainly the reason why the post was made

**Reply by Rubiidev-IPad** [reply]
i wonder who that person is...

**Reply by mariocraft987** [reply]
yup their prb gonna be fired

**Reply by raynepaws** [reply]
> i wonder who that person is...

you

**Reply by Rubiidev-IPad** [reply]
can you link the ai commits by me and my alt i forgor them, also your free to rewrite the ai things

**Reply by dumorando** [reply]
https://github.com/mariocraft987/bark-coding/commit/0351e66b37e4897df1fa5f8e7ed13e8f066bb731
https://github.com/mariocraft987/bark-coding/commit/a13f5ac83e344d9b07919d833817c6df4ee3b40c
theres alot more but heres the newer ones
<br />

![image](https://github.com/mariocraft987/bark-coding/assets/144301348/ec006939-becd-440a-97ec-983e7d484255)

**Reply by raynepaws** [reply]
> your free to rewrite the ai things

well that's kind of a lot, but i would take joy in doing that :)

**Reply by Rubiidev-IPad** [reply]
yea go ahead!

**Comment by raynepaws** [comment]
@Rubiidev-IPad so how much have you contributed to bark that *wasn't* ai?

**Reply by Rubiidev-IPad** [reply]
quite a bit, but it was just tweaks, new categories, and other stuff,

**Reply by dumorando** [reply]
i'm not trying to be rude here but all i've seen is readme changes and changes to the html like changing a number

**Reply by dumorando** [reply]
i apologize if weve came off as rude

**Reply by Rubiidev-IPad** [reply]
yea it was much smaller but still, on @Rubiidev-18, there was some big changes

**Reply by dumorando** [reply]
all the big changes were ai, or atleast things that change the user experience :confused:
